cmd/init.go

```
@@ -23,9 +24,9 @@ func init() {
 }
 
 func runInit(cmd *cobra.Command, args []string) error {
-	logger, err := micrologger.New(micrologger.DefaultConfig())
+	logger, err := micrologger.New(micrologger.Config{})
 	if err != nil {
-		return err
+		return microerror.Mask(err)
```

Lets please always mask errors.